src/gluonts/model/deepar/_network.py

```diff
@@ -425,6 +464,27 @@ def hybrid_forward(
         # need to mask possible nans and -inf
         loss = F.where(condition=loss_weights, x=loss, y=F.zeros_like(loss))
 
+        if self.alpha or self.beta:
+            # get accumulated outputs
+            outputs_raw = []
+            outputs_dropped = []
+            for state, state_info in zip(self.states, self.rnn.state_info()):
+                if 'state_index' in state_info:
```

Comment:
so the format is [dropped_h, c, h]?

Reply:
for each rnn layer, after accumulation, the final version of states will be [dropped_h, c, raw_h, [list of dropped_h], [list of raw_h]]
since there are 2 layers, the states of them will be concatenated.
so there should be totally 10 elements in the final states.

Reply:
the logic is:
for raw lstm, the states if [dropped_h, c]
after add zoneout with the option "preserve_raw_output=True", it will append raw_h to the states, then the states become [dropped_h, c, raw_h]
then, after add AccumulateStatesCell with option "index_list=[0, -1]", it will accumulate dropped_h and raw_h throughout timestamps and append them to states, so finally the states become [dropped_h, c, raw_h, [list of dropped_h], [list of raw_h]]

Reply:
10 elements?

Reply:
deepar has 2 rnn layers by default
every rnn layer has 5 elements, concatenating 2 layers gets 10.